keyboards/dz60/keymaps/via/rules.mk

```diff
@@ -0,0 +1,3 @@
+MOUSEKEY_ENABLE = no
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```

Comment:
```suggestion
LTO_ENABLE = yes
```

Reply:
In [via docs](https://caniusevia.com/docs/configuring_qmk) it says it should be `LINK_TIME_OPTIMIZATION_ENABLE`. I can see there are both `LINK_TIME_OPTIMIZATION_ENABLE` and `LTO_ENABLE` used around the code. Which one should I stick with?

Reply:
`LTO_ENABLE` is just an alias for `LINK_TIME_OPTIMIZATION_ENABLE`. The end result is the same.

https://github.com/qmk/qmk_firmware/blob/004ef3fad783832a5b61baa331b45ef5ce67fb9b/tmk_core/common.mk#L206-L215